terraform/deploy/app/orchestration_service.tf

```diff
@@ -22,7 +22,24 @@ module "ecs-fargate-task-definition" {
       "awslogs-stream-prefix" = "ecs"
     }
   }
-  environment = []
+  environment = [
+   {
+      name  = "orchestrationService.user_container_task_definition"
+      value = "${var.name_prefix}-ui-service"
+   },
+   {
+      name  = "orchestrationService.load_balancer_name"
+      value ="${var.name_prefix}-lb"
+   },
+    {
+      name  = "orchestrationService.aws_region"
+      value = var.region
+    },
+    {
+      name  = "orchestrationService.cognito_user_pool_id"
+      value = data.terraform_remote_state.aws_analytical_env_cognito.outputs.cognito.user_pool_id
```

Comment:
The `data.terraform_remote_state.aws_analytical_env_cognito` resource is not defined in the `terraform.tf.j2` causing the PR ci job to fail. Make sure to also add all resources you created in the `terraform.tf` into the `terraform.tf.j2` file.